src/@next/Link/LinkStyle.ts

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import { Icon } from '../Icon';
+import { Typography } from '../Typography';
+import { Blue } from '../utilities/colors';
+import { space2 } from '../utilities/spacing';
+import { LinkProps } from './Link';
+
+export const StyledLink = styled.a<LinkProps>`
+  display: flex;
+  align-items: center;
+  color: ${Blue.S99};
+
+  &[data-underline='false'] {
+    text-decoration: none;
+  }
+`;
+
+export const StyledTypography = styled(Typography)`
+  color: ${Blue.S99};
+
+  &:hover {
+    color: rgba(0, 86, 140, 0.9);
+  }
+
+  &:active {
+    color: ${Blue.S100};
+  }
+`;
+
+export const StyledIcon = styled(Icon)`
```

Comment:
I think we wouldn't need this too, can style them as the child of StyledLink

Reply:
you're right! let me update it